sn_client/src/file_apis.rs

```diff
@@ -315,7 +315,7 @@ impl Files {
             let missing_chunks: Vec<XorName> = chunks_info
                 .iter()
                 .filter_map(|expected_info| {
-                    if !retrieved_chunks.iter().any(|retrieved_chunk| {
```

Comment:
What's this change about? Is it supposed to be in this PR?

Reply:
that's a fix to a bug when throwing a bug during the data_with_churn tests.
The error supposed to contain a list of `missing chunks` , but the previous log acutally put the `fetched chunks` in.